src/interfaces/blobs.js

```diff
@@ -364,25 +371,28 @@ export default function ({url}) {
 						state: BLOB_STATE.ABORTED
 					};
 				case transformationFailed:
+					logger.log('debug', `case: ${op}, Error: ${payload.error}`);
 					return {
 						state: BLOB_STATE.TRANSFORMATION_FAILED,
 						modificationTime: moment(),
 						$set: {
 							'processingInfo.transformationError': payload.error
 						}
 					};
+				case transformedRecordFailed:
+					return {
+						modificationTime: moment(),
+						$push: {
+							'processingInfo.failedRecords': payload.transformedRecord
+						}
+					};
```

Comment:
There should be a new operation which is used to update the timestamp of the blob metadata and number of records. The operation `transformationDone` is sent when all records have been processed and no payload is needed.